docs/content/manual/release-specific/v1.2.4/test-instance-manager-npe.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Test instance manager NPE 
+---
+
+### Test step
+1. Create and attach a 1-replica volume.
+2. Create 2 snapshots with large amount of data so that rebuilding each snapshot would take some time.
+3. Disable the scheduling for the nodes so that there is one node could accept new replicas of the volume.
+4. Update the replica count to 2 for the volume and wait for the rebuilding start.
+5. While syncing the 1st snapshot file, create a directory with the name of **another snapshot meta file**. 
```

Comment:
Ok, so this is to simulate the receiver failing to create the file to cause the sender to close the sending server. 

Before step 6, should we delete the intensively created directory of the second snapshot meta file to make sure the following rebuilding is successful?

Reply:
No. We are intended to fail this rebuilding. And this messed rebuilding replica would be cleaned up after the failure hence we don't need to do anything here.